app/src/main/java/com/cpr3663/cpr_scouting_app/Match.java

```diff
@@ -410,6 +410,11 @@ public void start_Match() {
         but_MatchControl.setText(getResources().getString(R.string.button_start_teleop));
         but_MatchControl.setBackgroundColor(ContextCompat.getColor(this.getApplicationContext(), R.color.dark_yellow));
         but_MatchControl.setCompoundDrawablesWithIntrinsicBounds(0, 0, R.drawable.start_teleop, 0);
+
+        if (PreMatch.StartNote) {
+            Globals.EventLogger.LogEvent(Globals.EventList.getEventId("Auto_StartingNote"), 0, 0, true);
```

Comment:
There's not, i was wrong.  So, add this into Constants class near the others, and use it instead of the string.  eventPrevious can then just this as well.

   public static final int EVENT_ID_AUTO_STARTNOTE = 0;

Reply:
You mean for start note use the id not the string?

Reply:
Yes

Reply:
you don't event need to call "EventList.getEventId".  just use the constant.

Reply:
I don't use "EventList.getEventId" at all.